crates/burn-tensor/src/repr/operation.rs

```diff
@@ -16,6 +16,21 @@ use crate::{
 
 use super::{QuantizationParametersDescription, QuantizedTensorDescription};
 
+/// Custom operation in fusion stream, declaring it's inputs and outputs.
+#[derive(Clone, Debug, Hash, PartialEq, Serialize, Deserialize)]
+pub struct CustomOpDescription {
+    /// Input tensors used in this the custom operation.
```

Comment:
I would add an `id` field (a string for now). And maybe a `scalars` field too, but it's not important to have it now.